Editor/DefaultAppUtility.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.IO;
+using Microsoft.Win32;
+
+public static class DefaultAppUtility
```

Comment:
Please make this into an internal class, unless we want to open this API to users.

Reply:
Done